Makefile

```diff
@@ -47,8 +47,22 @@ fix: $(FIXERS)
 
 # END: lint-install ../bincapz
 
+SAMPLES_HASH=bdcb8c2e9bf557a0abe3e2b0144f437d456299b7
+out/samples-$(SAMPLES_HASH):
+	mkdir -p out
+	git clone https://github.com/chainguard-dev/bincapz-samples.git out/samples-$(SAMPLES_HASH).tmp
+	git -C out/samples-$(SAMPLES_HASH).tmp checkout $(SAMPLES_HASH)
+	for file in caddy.xz chezmoi.xz minio_x86_64.xz mongosh.xz neuvector_agent_aarch64.xz opa.xz ; do \
```

Comment:
Could this work to avoid hardcoding filenames?

```find <path> -name "*.xz" -exec xz -d {}```

Reply:
Implemented a variant of this in [`4ad123f` (#448)](https://github.com/chainguard-dev/bincapz/pull/448/commits/4ad123f821ebef10b812869682c6d1b38610c9a0).